app/views/admin/service_providers/index.html.erb

```diff
@@ -42,24 +42,51 @@
   </div>
 </div>
 
-<table class="usa-table">
-  <thead>
-    <tr>
-      <th>Organization</th>
-      <th>Name</th>
-      <th># of services</th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @service_providers.each do |service_provider| %>
-      <tr>
-        <td><%= service_provider.organization.name %></td>
-        <td><%= link_to service_provider.name, admin_service_provider_path(service_provider) %></td>
-        <td><%= service_provider.services.size %></td>
-      </tr>
+<div class="well">
+  <div class="field">
+    <%= label_tag :search_text, "Search by organization or name" %>
+    <%= text_field_tag :search_text, nil, class: "usa-input" %>
+    <% if @tags.size > 0 %>
+      <p>
+        Search by tag
+      </p>
+      <p class="tag-list">
+        <% @tags.order(:name).each do | tag | %>
+          <a href="#" class="search-tag-link" data-name="<%= tag.name %>">
+            <span class="usa-tag">
+              <%= tag.name %> (<%= tag.taggings_count %>)
```

Comment:
if I added a "mobile" tag to Service Provider and Service, the label on each shows 2, when I'd expect 1.

Is there a way to scope "taggings_count" ?